astropy/io/ascii/ui.py

```diff
@@ -220,7 +220,7 @@ def _guess(table, read_kwargs, format, fast_reader):
             reader.guessing = True
             return reader.read(table)
 
-        except (core.InconsistentTableError, ValueError, TypeError,
+        except (core.InconsistentTableError, ValueError, TypeError, AttributeError,
```

Comment:
Yeah I think that's fine.  I think it's worth opening an issue for though.  Maybe it could be done without *too* many try/excepts.  Just in a few key places.